website/content/en/docs/best-practices/observability-best-practices.md

```diff
@@ -0,0 +1,255 @@
+---
+title: "Operator Observability Best Practices"
+linkTitle: "Observability Best practices"
+weight: 6
+description: This guide describes the best practices concepts for adding Observability to operators.
+---
+
+## Operator Observability Best Practices
+
+In this document we will outline what operators require in order to meet the "Deep Insights" capability level and provide best practices and examples for creating metrics, recording rules and alerts. It is based on the general guidelines in [Operator Capability Levels](https://sdk.operatorframework.io/docs/overview/operator-capabilities/).
+
+**Note:** For technical documentation of how to add metrics to your operator, please read the [Metrics](https://book.kubebuilder.io/reference/metrics.html) section of the Kubebuilder documentation.
+
+### Deep Insights capability level requirements
+
+1. **Health and Performance metrics** for all of the operator components - Implemented based on the guidelines below.
+2. **Metrics Documentation** - All metrics should have documentation.
+3. **Alerts** for when things are not working as expected for each of the operator's components - Implemented based on the guidelines below.
+4. **Alerts Runbooks** - Each alert MUST include a runbook_url annotation and a runbook.
```

Comment:
I guess this refers to the link for runbook. For a prometheus alert, it can be attached with such `runbook_url` so that when the alert triggered, the people on-call can go to the link and see the procedure to resolve the issue.

e.g.:
<img width="1407" src="https://user-images.githubusercontent.com/18136486/205117624-f2fdcfa0-f977-480a-94d1-7060980171e0.png">

Reply:
Yes, I updated it to be optional.